cmd/init.go

```diff
@@ -80,6 +81,11 @@ to quickly create a Cobra application.`,
 		// name of the cloud region to provision resources when resources are region-specific
 		region, _ := cmd.Flags().GetString("region")
 		viper.Set("aws.region", region)
+		// propagate it to local environment
+		err = os.Setenv("AWS_REGION", region)
+		if err != nil {
+			log.Panicf("unable to set environment variable AWS_REGION, error is: %v", err)
+		}
```

Comment:
This need to be a required flag, not a variable. Logs is also not reflecting it.

Reply:
`os.Setenv` due to some weird ways go fork shells, doesn't produce the expected behavior, specially for terraform.

Reply:
it was actually there:
```
	initCmd.Flags().String("region", "", "the region to provision the cloud resources in")
	err = initCmd.MarkFlagRequired("region")
	if err != nil {
		log.Panic(err)
	}
```

Reply:
As this is a flag, the message should refer to a missing flag, or a wrong valuer of a flag. Not a variable, variables means a "Environment Variable" on this context for most users.

Reply:
Ahh, I see.. it is a error setting.. never mind. Approving.